src/legacy/core_plugins/metrics/public/components/vis_editor.js

```diff
@@ -132,7 +132,7 @@ class VisEditor extends Component {
 
     if (model) {
       return (
-        <div className="tvbEditor">
+        <div className="tvbEditor" data-test-subj="tvbEditor">
```

Comment:
Might want to rename `tvbEditor` to `tvbVisEditor` so it has a consistent prefix with `tvbVisGaugeContainer`. 

The prefix consistency for TSVB is varying with prefixes such as `tsvb` also used. Not reason to introduce a new prefix